addons/netfox/state-synchronizer.gd

```diff
@@ -30,16 +30,37 @@ func _ready():
 func _after_tick(_dt, tick):
 	if is_multiplayer_authority():
 		# Submit snapshot
-		var state: Dictionary = PropertySnapshot.extract(_props)
-		_submit_state.rpc(state, tick)
+		var local_state: Dictionary = PropertySnapshot.extract(_props)
+		update_last_state_cache(local_state, tick)
```

Comment:
I haven't tested this class at all, nor am I familiar with it. But it should work since rollback synchronizer works with state serialization. The logic is quite simple, so I doubt I missed something.